pom.xml

```diff
@@ -647,10 +647,94 @@
                   <include>io.github.openfeign.querydsl:*</include>
                 </includes>
                 <excludes>
+                  <exclude>*:querydsl-examples</exclude>
                   <exclude>*:querydsl-example-*</exclude>
-                  <exclude>*:querydsl-benchmark</exclude>
+                  <exclude>*:querydsl-jpa-spring</exclude>
```

Comment:
I guess you need to ignore `<exclude>*:querydsl-sql-spring</exclude>` as well then.

I am wondering whether adding this would solve the whole Spring Boot/Data dependency issue (#275) as nothing in the generated `querydsl-bom/pom.xml` would then depend on Spring.

Spring Data JPA doesn't depend on the BOM, they have direct dependencies on the QueryDSL modules: https://github.com/search?q=repo%3Aspring-projects%2Fspring-data-jpa+querydsl+language%3A%22Maven+POM%22&type=code&l=Maven+POM
Spring Boot has a dependency on `querydsl-bom`: https://github.com/spring-projects/spring-boot/blob/3.2.x/spring-boot-project/spring-boot-dependencies/build.gradle#L1300

So unless Spring Data/Boot adds an additional dependency on querydsl-\*-spring (which they will never do due to cyclic dependencies), or unless some developer has a dependency on both querydsl-\*-spring and Spring Boot/Data with non-matching Spring versions, all should be fine IMO.

Reply:
> I am wondering whether adding this would solve the whole Spring Boot/Data dependency issue (https://github.com/OpenFeign/querydsl/issues/275) as nothing in the generated querydsl-bom/pom.xml would then depend on Spring.

I have no idea, I don't have any clue what the problem is.